components/mixins/experience-event/experienceevent-alert-impressions.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-alert-impressions",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Alert Impressions",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": [
+      "retail",
+      "financial_services",
+      "travel_and_hospitality",
+      "health_and_life_sciences",
+      "public_sector"
+    ]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Alert information such as ad impressions and ad clicks. ",
```

Comment:
We need to understand a little bit more on the alerts and how are they related to marketing ?

Reply:
Per our conversation, this has been updated to "alerts"